dll/win32/srclient/srclient_main.c

```diff
@@ -29,40 +29,33 @@
 
 BOOL
 WINAPI
-SRSetRestorePointA(PRESTOREPOINTINFOA pRestorePtSpec, PSTATEMGRSTATUS pStateMgrStatus)
+SRSetRestorePointA(PRESTOREPOINTINFOA pRestorePtSpec, PSTATEMGRSTATUS pSMgrStatus)
 {
-    RESTOREPOINTINFOW RPInfoW;
-
     DPRINT1("SRSetRestorePointA is unimplemented\n");
 
-    if (!pRestorePtSpec)
+    if (!pSMgrStatus)
         return FALSE;
 
-    RPInfoW.dwEventType = pRestorePtSpec->dwEventType;
-    RPInfoW.dwRestorePtType = pRestorePtSpec->dwRestorePtType;
-    RPInfoW.llSequenceNumber = pRestorePtSpec->llSequenceNumber;
-
-    if (!MultiByteToWideChar( CP_ACP, 0, pRestorePtSpec->szDescription, -1, RPInfoW.szDescription, MAX_DESC_W))
```

Comment:
Why is this an improvement? The old version handled `szDescription` but this does not seem to?